### ISSUE TITLE
Should we iframe tab popups even in the museum display

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/OZtree#46]
E.g. it might be possible to create our *own* wikipedia frames, which encapsulate the wikipedia page but also turn off pinch-to-zoom. That way we don't need to inject e.g. the wikipedia css into the main OneZoom page, which has always struck me as a bit hairy.

URLs could be of the form www.onezoom.org/iframe/wiki.html?Qid=xxx&lang=xxx

Other advantages: we could *possibly* do the same for EoL / IUCN / etc pages (perhaps not for the museum display, but simply displaying the page with links and a BASE tag set), which will get around https://github.com/jrosindell/OneZoomComplete/issues/381. With our own pages, this be a potential solution to https://github.com/OneZoom/OZtree/issues/21 and also to the issue of moving from the sponsor_node page to the sponsor_leaf page.

**Comment by jrosindell** [comment]
Interesting idea, but I think not necessary for the display which will focus on wiki stuff only.  It might be useful for the main website

**Comment by hyanwong** [comment]
Yes, but if we do it for one version, it's easiest to do it for all, and not have to have bespoke code for injecting html into the OneZoom web page. It also might help for the sponsorship tab. 

Of course, for the museum display we would just turn off EoL, IUCN etc tabs anyway, and it should look *visually* the same. But I think the whole thing might be cleaner, if it's possible to do.

**Comment by hyanwong** [comment]
Remember to remove the line after "" in treeviewer/layout.html

**Comment by hyanwong** [comment]
If we want to be *really* nice to the user, we could re-enable zooming within the iframe only, e.g. as in the response to https://stackoverflow.com/a/42053317/2667066

**Comment by hyanwong** [comment]
Even better, we should probably implement panzoom (http://timmywil.github.io/jquery.panzoom/) within our embedded pages
